flow/templates/base_status.jinja

```diff
@@ -31,6 +34,16 @@
 {{ field_progress_bar | format(label[0], label[1] | draw_progressbar(jobs | length)) }}
 {% endfor %}
 {% endblock%}
+{% block operation_summary %}
+
+{% set field_head = field_operation_title + '%s' %}
+{{ field_head | format('operation', 'number of eligible jobs') }}
+{{ field_head | format('-' * (column_width_operation+4), '-' * column_width_num_jobs) }}
+{% for op, n_jobs in op_counter.items() %}
+{% set x = op | length %}
```

Comment:
Can we name this something meaningful like `op_length`?

Reply:
👍